wpeverest/everest-forms
# Feature - Import entries in our form using csv

### All Submissions:






### Changes proposed in this Pull Request:



Previously, there is no feature to import csv data to our entries. This PR add this feature.

### How to test the changes in this Pull Request:

1. Go to Everest Forms > Tools > Import.
2. Select form and upload csv file and click **Map CSV**.
3. Add fields of the form to map form fields with csv columns. And Click on **Import Entries**.
4. Check Entries whether csv data are imported or not?
5. Csv file must contain huge data.

### Types of changes:




### Other information:




### Changelog entry

> Feature - Import entries in our form using csv
